app/partials/project-card.cjsx

```diff
@@ -12,6 +12,9 @@ counterpart.registerTranslations 'en',
 module.exports = React.createClass
   displayName: 'ProjectCard'
 
+  projectOwner: ->
+    apiClient.type(@props.project.links.owner.type).get(@props.project.links.owner.id)
```

Comment:
Hm, it hasn't been re-requesting for me.

What're you planning on doing for the avatar?